public/app/types/explore.ts

```diff
@@ -322,6 +322,8 @@ export interface QueryTransaction {
 
 export type RangeScanner = () => RawTimeRange;
 
+export type ResultGetter = (result: any, transaction: QueryTransaction, allTransactions: QueryTransaction[]) => any;
```

Comment:
Do we have not have types for the result?

Reply:
think it should be DataQueryResponse

Reply:
no, as you do res.data, TimeSeries[] | [TableData] | any
https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/types/datasource.ts#L6

The any is for LogsStream thing that is not un grafana-ui yet (and should be replaced with LogsModel I think).  Feels like we need a type for this like:

DataQueryData = TimeSeries[] | [TableData] | any; 

hate the typename :(

Reply:
I've added typings for result, could you have a look again?